docs/stub-attribution.rst

```diff
@@ -72,8 +72,10 @@ the page template where an experiment exists:
 .. code-block:: html
 
     {% block stub_attribution %}
+      <!--[if !lte IE 8]><!-->
```

Comment:
Is it worth putting the conditional comments around the the outside of the sub_attribution block so they can't be missed?

Reply:
`{% if self.stub_attribution() %} <!--[if !lte IE 8]><!--> {% endif %}`

Reply:
Not quite sure I follow this suggestion, but we don't want to exclude old IE's from regular stub attribution, only our experiment code.